artifacts/definitions/Windows/Forensics/SRUM.yaml

```diff
@@ -25,17 +24,29 @@ parameters:
   - name: ExecutionGUID
     default: "{5C8CF1C7-7257-4F13-B223-970EF5939312}"
     type: hidden
-
-
+  - name: NetworkUsageGUID
+    default: "{973F5D5C-1D90-4944-BE8E-24B94231A174}"
+    type: hidden
+  - name: Upload 
+    description: Select to Upload the SRUM database file 'srudb.dat'
+    type: bool 
+    
 sources:
   - name: Upload
-    queries:
-      - SELECT upload(file=SRUMLocation, accessor=accessor) AS Upload
-        FROM scope()
+    query: |
+        SELECT * from if(condition=Upload,
```

Comment:
You could use a precondition on the source to achieve the same thing

Reply:
I'm not sure how that one goes. Could you modify or show me an example? Is it similar to preconditions related to the OS?

Reply:
yes exactly - a precondition is just a query that is run before the main query so something  like:

```sql
SELECT * FROM scope() WHERE Upload 
```

It doesnt really matter what you select - if it returns more than 1 row the query will run.